image-customize

```diff
@@ -189,10 +189,7 @@ class BuildAction(ActionBase):
         machine.execute(f"su builder {opt_build_options} /var/tmp/mkbuild.sh")
 
         # build binaries
-        machine.execute("cd /var/tmp/build; su builder -c extra-x86_64-build", timeout=1800)
-
-        # install packages
-        machine.execute("pacman -U --noconfirm /var/tmp/build/*.pkg.tar.zst")
+        machine.execute("cd /var/tmp/build; makechrootpkg -r /var/lib/archbuild/cockpit -U builder -- -i", timeout=1800)
```

Comment:
-i tells makepkg to install the build packages, however this is run **in the chroot** not outside! *faceplam* :face_with_head_bandage:

Reply:
As alternative we could call `makepkg --packagelist` and use that to install (it needs to be run from the PKGBUILD directory) so we don't hardcode the package extension here